src/Knp/Snappy/AbstractGenerator.php

```diff
@@ -21,6 +21,10 @@ abstract class AbstractGenerator implements GeneratorInterface, LoggerAwareInter
 {
     use LoggerAwareTrait;
 
+    protected const ALLOWED_PROTOCOLS = ['file'];
+
+    protected const WINDOWS_LOCAL_FILENAME_REGEX = '/^[a-z]:(?:[\\\\\/]?(?:[\w\s!#()-]+|[\.]{1,2})+)*[\\\\\/]?/i';
```

Comment:
Is the first letter always lower case? C:` won't happen?

Reply:
No, but the regular expression is case-insensitive, so no need of adding `A-Z`.

Reply:
`/i`.... indeed.